docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<p>I think it should be <code>[i-excl_zone : i+excl_zone+1]</code> and not <code>[i-excl_zone+1 : i+excl_zone]</code></p><p> </p><p>One should probably define <code>excl_zone</code> first rather than after</p><p> </p><p>There is a typo in <code>(0, i-excl_zon)</code>) (i.e., you are missing an "e" in "zone")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
@seanlaw 
Could you confirm this? I think again and I think `i-excl_zone+1 : i+excl_zone` sounds correct to me. For instance, let's assume `i=3` and `excl_zone=2`. In that case, the subsequence starts at index 1 is okay and not inside the exclusion area. Is that correct? 
If I remember correctly, in one of previous PRs, you mentioned that the exclusion is `[i-L+1: i+L]`. 

Please let me know what you think.

Reply:
<p>No, for <code>i=3</code>  and <code>excl_zone=2</code> , all distances, <code>D</code>,  computed between <code>T[3:3+m]</code>  and subsequences <code>T[1:1+m]</code> , <code>T[2:2+m]</code> , <code>T[3:3+m]</code> , <code>T[4:4+m]</code> , <code>T[5:5+m]</code>  are all set to <code>np.inf</code>  because they are all within the exclusion zone (i.e., <code>D[1:6] = np.inf</code> (numpy indexing) for <code>i=3</code> ). </p>

Reply:
Thanks for clarification! I got it. I am going to change the code accordingly.